src/Assets/TestProjects/PortableTool/consoledemo.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp1.1;netcoreapp2.0</TargetFrameworks>
+    <AssemblyName>consoledemo</AssemblyName>
+    <PackAsTool>true</PackAsTool>
```

Comment:
@KathleenDollard this part need your review. The experience is, use property `PackAsTool` to mark, and then user run dotnet pack to create a tool.

The entry point and command name will use MSBuild well known property `TargetFileName` as default (The file name of the primary output file for the build https://msdn.microsoft.com/en-us/library/c02as0cs.aspx)

The user can override it by adding explicit property `ToolCommandName` `ToolEntryPoint`

Reply:
Let me know when you get in. I'd like to touch base on this. I think I get it, but want to check.

Reply:
Add Nuget package name override here